themes/bootstrap3/templates/layout/layout.phtml

```diff
@@ -86,12 +86,14 @@
       $translations = !isset($this->renderingError) ? $this->render('js-translations.phtml') : '{}';
       $dsb = DEFAULT_SEARCH_BACKEND;
       $cspNonce = $this->cspNonce();
+      $searchId = $this->layout()->searchId;
       $appendScripts[] = <<<JS
 VuFind.path = '{$root}';
 VuFind.defaultSearchBackend = '{$dsb}';
 VuFind.addTranslations({$translations});
 VuFind.addIcons({$icons});
 VuFind.setCspNonce('{$cspNonce}');
+VuFind.setCurrentSearchId($searchId);
```

Comment:
Should we be more explicit about what happens when searchId is not set? Right now, it doesn't seem to hurt anything, but we end up calling setCurrentSeachId without a parameter, which I don't think is exactly expected behavior. Maybe we should explicitly pass a null here, or else add a default value to the method's parameter in common.js.

Reply:
Oops, yes. I added json_encode so that it can handle any value just in case.